rotkehlchen/db/cache.py

```diff
@@ -1,28 +1,35 @@
-from rotkehlchen.errors.serialization import DeserializationError
 from rotkehlchen.types import Timestamp
 from rotkehlchen.utils.mixins.enums import Enum
 
 
 class DBCache(Enum):
-    """It contains all the values that can be stored in the `key_value_cache` table of the DB"""
+    """It contains all the exact keys that can be stored in the `key_value_cache` table"""
     LAST_BALANCE_SAVE = 'last_balance_save'
     LAST_DATA_UPLOAD_TS = 'last_data_upload_ts'
     LAST_DATA_UPDATES_TS = 'last_data_updates_ts'
     LAST_OWNED_ASSETS_UPDATE = 'last_owned_assets_update'
     LAST_EVM_ACCOUNTS_DETECT_TS = 'last_evm_accounts_detect_ts'
     LAST_SPAM_ASSETS_DETECT_KEY = 'last_spam_assets_detect_key'
     LAST_AUGMENTED_SPAM_ASSETS_DETECT_KEY = 'last_augmented_spam_assets_detect_key'
+    LAST_EVENTS_PROCESSING_TASK_TS = 'last_events_processing_task_ts'
+    LAST_PRODUCED_BLOCKS_QUERY_TS = 'last_produced_blocks_query_ts'
+    LAST_WITHDRAWALS_EXIT_QUERY_TS = 'last_withdrawals_exit_query_ts'
 
-    @classmethod
-    def deserialize(cls: type['DBCache'], value: str) -> 'DBCache':
-        """The method to deserialize a string to an enum object"""
-        try:
-            return getattr(cls, value.upper())
-        except AttributeError as e:
-            raise DeserializationError(f'Failed to deserialize {cls.__name__} value {value}') from e  # noqa: E501
 
+def maybe_get_cache_enum(value: str) -> DBCache | str:
+    """A helper function to convert string to `DBCache` enum, if available"""
+    if value in {enum.value for enum in DBCache}:
+        return getattr(DBCache, value.upper())
+    else:
+        return value
 
-def serialize_cache_for_api(cache: dict[DBCache, Timestamp]) -> dict[str, Timestamp]:
+
+def serialize_cache_name(name: DBCache | str) -> str:
+    """A serialization function to conver possible `DBCache` enum to string"""
+    return name.value if isinstance(name, DBCache) else name
```

Comment:
after the change I believe you will be using this only in one place. Remove this function

Reply:
This is needed in two functions (`get_cache` and `set_cache`). But, I removed the other two functions (`serialize_cache_for_api` and `maybe_get_cache_enum`)